neurogym/envs/native/annubes.py

```diff
@@ -5,68 +5,23 @@
 from neurogym.core import TrialEnv
 from neurogym.utils import spaces
 
+# A reserved keyword for catch trials.
+CATCH_KEYWORD: str = "catch"
+CATCH_SESSION_KEY: tuple[str] = (CATCH_KEYWORD,)
+
+# The upper and lower bounds for the catch probability used in the max_sequential satisfiability checks.
+MIN_CATCHPROB: float = 1e-6
+MAX_CATCHPROB: float = 1 - 1e-6
```

Comment:
Out of curiosity, why did you take these out of AnnubesEnv?

Reply:
I mainly wanted to draw the reserved keywords out. Those are not values a user should be touching. I guess I pulled these out along without giving it too much thought. I think there's an argument to be made either way and don't actually have a strong preference.